build.sh

```diff
@@ -58,7 +61,11 @@ build_blake2bf() {
 EOF
 
   if [[ "$OSTYPE" == "linux-gnu" ]];  then
-    cd "$SCRIPTDIR/blake2bf/$( arch )"
+    if [[ $(uname -m) == 'aarch64' ]]; then
```

Comment:
I am a bit confused by this:

- if `$(uname -m) == 'aarch64'` then `arm64` dir
- if `$(uname -m) == 'arm64'` then `aarch64` dir

it seems clearer to simply swap the dir names, or am I missing something?

Reply:
this is to override - on m1 besu self hosted runner, I found the `$(arch)` call returned an invalid value. Instead, I catch early that if `uname -m` is aarch64, then we need to use the arm64 sources.